src/pages/articles-details/articles-details.html

```diff
@@ -10,9 +10,9 @@
     <div padding *ngIf="articles">
       <h2>{{articles.title}}</h2>
       <h3>{{articles.published | amFromUnix | amDateFormat:'LL'}}</h3>
-      <ion-row>
-        <ion-col>
-          <ion-slides *ngIf="articles.images" loop="{{articles.images.length > 1 ? true:false}}" autoplay="3500" pager="{{articles.images.length > 1 ? true:false}}">
+      <ion-row *ngIf="articles.images"> 
+        <ion-col *ngIf="articles.images.length > 1">
```

Comment:
I don't understand how could it works, if images.length == 1 don't display any image ?